openfl/component/director/workspace.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Workspace module."""
+
+import os
+import shutil
+from pathlib import Path
+
+
+class AggregatorWorkspace:
```

Comment:
I suggest moving this away from the components package, to the utilities probably

Reply:
Moved to utilities/workspace.py